Source/CarthageKit/BuildArguments.swift

```diff
@@ -84,6 +84,11 @@ public struct BuildArguments {
 			args += [ "BITCODE_GENERATION_MODE=\(bitcodeGenerationMode.rawValue)" ]
 		}
 
+		if let carthagePath = NSBundle.mainBundle().executablePath
+			where NSBundle.mainBundle().executableURL?.lastPathComponent?.lowercaseString.rangeOfString("carthage") != nil {
```

Comment:
My thinking was around people who had multiple versions installed, maybe a beta and a release version. If it's not considered helpful I can make the check just for `carthage`